[INFO   ] Logger: Record log in /home/kerb3rosz/Documents/Munka/darkdevops/codebase/programozas/python/gui/10.configuracio/logs/kivy_20-05-23_3.txt
[INFO   ] Kivy: v1.11.1
[INFO   ] Kivy: Installed at "/usr/local/lib/python3.7/dist-packages/kivy/__init__.py"
[INFO   ] Python: v3.7.7 (default, Apr  1 2020, 13:48:52) 
[GCC 9.3.0]
[INFO   ] Python: Interpreter at "/usr/bin/python"
[INFO   ] Factory: 184 symbols loaded
[INFO   ] ImageLoaderFFPy: Using ffpyplayer 4.3.1
[INFO   ] Image: Providers: img_tex, img_dds, img_sdl2, img_pil, img_ffpyplayer, img_gif 
[INFO   ] Window: Provider: sdl2(['window_egl_rpi'] ignored)
[INFO   ] GL: Using the "OpenGL" graphics system
[INFO   ] GL: Backend used <sdl2>
[INFO   ] GL: OpenGL version <b'3.0 Mesa 19.3.3'>
[INFO   ] GL: OpenGL vendor <b'Intel Open Source Technology Center'>
[INFO   ] GL: OpenGL renderer <b'Mesa DRI Intel(R) HD Graphics 630 (Kaby Lake GT2) '>
[INFO   ] GL: OpenGL parsed version: 3, 0
[INFO   ] GL: Shading version <b'1.30'>
[INFO   ] GL: Texture max size <16384>
[INFO   ] GL: Texture max units <32>
[INFO   ] Window: auto add sdl2 input provider
[INFO   ] Window: virtual keyboard not allowed, single mode, not docked
[INFO   ] Text: Provider: sdl2
[INFO   ] ProbeSysfs: device match: /dev/input/event7
[INFO   ] MTD: Read event from </dev/input/event7>
[INFO   ] Base: Start application main loop
[INFO   ] GL: NPOT texture support is available
[INFO   ] Base: Leaving application in progress...
[WARNING] stderr: Traceback (most recent call last):
[WARNING] stderr:   File "10.legorduloMenu.py", line 17, in <module>
[WARNING] MTD: Unable to open device "/dev/input/event7". Please ensure you have the appropriate permissions.
[WARNING] stderr:     TesztApp().run()
[WARNING] stderr:   File "/usr/local/lib/python3.7/dist-packages/kivy/app.py", line 855, in run
[WARNING] stderr:     runTouchApp()
[WARNING] stderr:   File "/usr/local/lib/python3.7/dist-packages/kivy/base.py", line 504, in runTouchApp
[WARNING] stderr:     EventLoop.window.mainloop()
[WARNING] stderr:   File "/usr/local/lib/python3.7/dist-packages/kivy/core/window/window_sdl2.py", line 747, in mainloop
[WARNING] stderr:     self._mainloop()
[WARNING] stderr:   File "/usr/local/lib/python3.7/dist-packages/kivy/core/window/window_sdl2.py", line 479, in _mainloop
[WARNING] stderr:     EventLoop.idle()
[WARNING] stderr:   File "/usr/local/lib/python3.7/dist-packages/kivy/base.py", line 339, in idle
[WARNING] stderr:     Clock.tick()
[WARNING] stderr:   File "/usr/local/lib/python3.7/dist-packages/kivy/clock.py", line 591, in tick
[WARNING] stderr:     self._process_events()
[WARNING] stderr:   File "kivy/_clock.pyx", line 384, in kivy._clock.CyClockBase._process_events
[WARNING] stderr:   File "kivy/_clock.pyx", line 414, in kivy._clock.CyClockBase._process_events
[WARNING] stderr:   File "kivy/_clock.pyx", line 412, in kivy._clock.CyClockBase._process_events
[WARNING] stderr:   File "kivy/_clock.pyx", line 167, in kivy._clock.ClockEvent.tick
[WARNING] stderr:   File "/usr/local/lib/python3.7/dist-packages/kivy/uix/floatlayout.py", line 128, in do_layout
[WARNING] stderr:     c.center = x + value[0] * w, y + value[1] * h
[WARNING] stderr: TypeError: 'float' object is not subscriptable
